tests/test_date.nit

```diff
@@ -0,0 +1,13 @@
+import date
```

Comment:
There is nothing in this file that could not be tested as a nitunit.

The "it will grow latter" argument is just invalid.

Please remove it.